tests/scanner/test_data/fake_retention_scanner_data.py

```diff
@@ -0,0 +1,101 @@
+# Copyright 2018 The Forseti Security Authors. All rights reserved.
```

Comment:
I'd suggest copying the style of the other files in tests/scanner/test_data, where (as much as possible) the test resources are just defined as dictionaries or lists of dictionaries, setting all dictionary keys in the one statement, rather than one key at a time.

Maybe have some common Lifecycle dictionaries as constants that can be shared in different tests